docs/datamodel/constraints.rst

```diff
@@ -44,7 +44,7 @@ the length of a string.
       constraint expression on (len(__subject__) < 25);
```

Comment:
Can you just change this to `<= 25` so we don't have different numbers?

Reply:
Of course! I changed it to <= 25 and updated the max_len_value condition to 25 as well, otherwise they wouldn't match.